tooling/cli/lib/command-modules/authenticate.js

```diff
@@ -39,10 +39,10 @@ exports.handler = async function (argv) {
   if (instances.length) {
     console.log('Initializing project...');
     await initProject(process.cwd(), instances);
-  }
 
-  const pkgManager = /yarn/.test(process.env.npm_execpath) ? 'yarn' : 'npm';
+    const pkgManager = /yarn/.test(process.env.npm_execpath) ? 'yarn' : 'npm';
 
-  console.log('\nReady to get started? Just run:\n');
-  console.log(`$ ${pkgManager} run dev`);
+    console.log('\nReady to get started? Just run:\n');
```

Comment:
If we're moving these in the `if` block, think it'd be helpful to have an `else` to show an error message re: the failure to auth/init instances instead of the command script completing quietly.